apps/05Mastermind.js

```diff
@@ -89,10 +82,21 @@ function mastermind(guess) {
 
 function getPrompt() {
     prompt.get(['guess'], function (error, result) {
-        console.log( mastermind(result['guess']) );
+        mastermind(result['guess']);
         if (!win) {
```

Comment:
Hey Matt! Looks like "should be able to detect a win" test did not pass. Is it passing on your end?